qiskit/user_config.py

```diff
@@ -138,6 +139,72 @@ def read_config_file(self):
                 self.settings['num_processes'] = num_processes
 
 
+def set_config(key, value, section=None, file_path=None):
+    """Adds or modifies a user configuration
+
+    It will add the configuration in either the default location
+    ~/.qiskit/settings.conf or the value of file argument.
```

Comment:
I would probably say in the currently configured location since if `QISKIT_SETTINGS` is set this won't be either either.